troposphere/templates/no_user.html

```diff
@@ -23,8 +23,8 @@
             </div>
             {% endif %}
             <h1>Your account does not currently have access to Atmosphere.</h1>
-            <p>If you would like to request access, you can do so through the <a href="https://user.iplantcollaborative.org/dashboard/">{{ ORG_NAME }} User Management Portal</a> by clicking the 'Request Access' button next to the Atmosphere service.</p>
-            <p>You can also find instructions <a href="http://cyverse.org/learning-center/manage-account#AddAppsServices">on the website</a>.</p>
+            <p>If you would like to request access, you can do so through the <a href="{{USER_PORTAL_LINK}}">{{USER_PORTAL_LINK_TEXT}}</a> by clicking the 'Request Access' button next to the Atmosphere service.</p>
+            <p>You can also find instructions <a href="{{ACCOUNT_INSTRUCTIONS_LINK}}">on the website</a>.</p>
             <p>If you have already requested access, then you must wait until your request is approved before you can use Atmosphere.</p>
             <p>If you have already been approved, you may be seeing this page because your session has expired. Click below to attempt another login.</p>
             <a class="submitButton" href="/logout?cas=true">
```

Comment:
Now that this page is so specific, I would like to see a future iteration that removes L29-30.

We would also need to upgrade the 'logic' to determine between 'No User' and 'User who has expired' and fire off to corresponding page.